packages/faceted-search/src/components/Badges/BadgeFaceted/BadgeFaceted.component.js

```diff
@@ -20,14 +20,6 @@ const theme = getTheme(cssModule);
 
 const findOperatorByName = name => operator => name === operator.name;
 
-const getOverlays = (initialOperatorOpened, initialValueOpened, operators) => {
-	if (operators.length < 2 && initialOperatorOpened) {
-		// To open the value just after the selection of the type
-		return useBadgeOverlayFlow(false, true);
-	}
-	return useBadgeOverlayFlow(initialOperatorOpened, initialValueOpened);
-};
-
```

Comment:
This is considered an error because it's a call of hook outside of a component.
Here we set initial values depending on a condition, let's move that directly to the component